packages/fuel-indexer-macros/src/decoder.rs

```diff
@@ -868,7 +871,7 @@ impl From<ObjectDecoder> for TokenStream {
                                         Self::TYPE_ID,
                                         self.to_row(),
                                         serialize(&self.to_row())
-                                    ).await;
+                                    ).await.expect(&format!("Entity::save for {} failed.", stringify!(#ident)));
```

Comment:
Same comment here:

- I think here we should be logging the error messages (and `.expect`) as close to the actual culprit call as possible
- In this case the culprit of this error would be `fuel_indexer::database::Database:: put_object`
  - If the error is logged/handled there, we don't need to handle it here